CodeSniffer.php

```diff
@@ -1933,6 +1933,10 @@ public static function isCamelCaps(
         $public=true,
         $strict=true
     ) {
+        if ('' === $string) {
```

Comment:
Because this is coding standard used in this library. I know that Yoda-style is highly appreciated in Symfony components though. Not really sure why and how it all started.